sonar-html-plugin/src/main/java/org/sonar/plugins/html/checks/sonar/LangAttributeCheck.java

```diff
@@ -36,6 +37,21 @@ private static boolean isHtmlTag(TagNode node) {
   }
 
   private static boolean hasLangAttribute(TagNode node) {
-    return node.hasProperty("lang") || node.hasProperty("xml:lang");
+    return node.hasProperty("lang")
+      || node.hasProperty("xml:lang")
+      || hasLangAttributeFromWordPress(node);
+  }
+
+  /**
+   * Using WordPress, HTML attributes can be set using the php function `language_attributes`
+   */
+  private static boolean hasLangAttributeFromWordPress(TagNode node) {
+    for (Attribute attribute : node.getAttributes()) {
+      String attributeName = attribute.getName();
+      if (attributeName.contains("?php") && attributeName.contains("language_attributes")) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
This is a matter of preference of implementation, but it's also possible to use Java 8 streams for this logic as follows:
```suggestion
    return node.getAttributes().stream()
      .map(Attribute::getName)
      .anyMatch(attributeName -> attributeName.contains("?php") && attributeName.contains("language_attributes"));
```
Up to you to keep as is or update :)

Reply:
I chose the stream version ;) Thanks for the java refresher :)